### ISSUE TITLE
get panic while initializing (v1.10.0)

### ISSUE THREAD

**Issue by justforfunya** [ava-labs/avalanchego#1440]
**Describe the bug**
I get a panic while starting up

```
[04-28|12:40:48.031] INFO <X Chain> avm/vm.go:275 address transaction indexing is disabled
[04-28|12:40:48.032] INFO chains/manager.go:757 creating proposervm wrapper {"activationTime": "[09-22|21:00:00.000]", "minPChainHeight": 793005, "minBlockDelay": "1s"}
[04-28|12:40:48.032] INFO <X Chain> snowman/transitive.go:89 initializing consensus engine
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xcba826]

goroutine 1949 [running]:
github.com/ava-labs/avalanchego/vms/proposervm.(*VM).getPostForkBlock(0xc0233a0d00, {0x1c08748, 0xc0000440c8}, {0x2a, 0xa9, 0x2c, 0x79, 0xb7, 0xb, 0x46, ...})
        /home/runner/work/avalanchego-internal/avalanchego-internal/vms/proposervm/vm.go:665 +0xa6
github.com/ava-labs/avalanchego/vms/proposervm.(*VM).parsePostForkBlock(0xc0233a0d00, {0x1c08748, 0xc0000440c8}, {0xc0156086c0?, 0x18?, 0xc001214c00?})
        /home/runner/work/avalanchego-internal/avalanchego-internal/vms/proposervm/vm.go:608 +0xb9
github.com/ava-labs/avalanchego/vms/proposervm.(*VM).ParseBlock(0xc0233a0d00, {0x1c08748, 0xc0000440c8}, {0xc0156086c0, 0x215, 0x240})
        /home/runner/work/avalanchego-internal/avalanchego-internal/vms/proposervm/vm.go:266 +0x45
github.com/ava-labs/avalanchego/vms/metervm.(*blockVM).ParseBlock(0xc005146fc0, {0x1c08748?, 0xc0000440c8?}, {0xc0156086c0?, 0x215?, 0x240?})
        /home/runner/work/avalanchego-internal/avalanchego-internal/vms/metervm/block_vm.go:112 +0xde
github.com/ava-labs/avalanchego/snow/engine/snowman/bootstrap.(*parser).Parse(0xc00f47e580, {0x1c08748?, 0xc0000440c8?}, {0xc0156086c0?, 0xc011b498cb?, 0x0?})
        /home/runner/work/avalanchego-internal/avalanchego-internal/snow/engine/snowman/bootstrap/block_job.go:33 +0x42
github.com/ava-labs/avalanchego/snow/engine/common/queue.(*state).GetJob(0xc016a5fa70, {0x1c08748, 0xc0000440c8}, {0x2a, 0xa9, 0x2c, 0x79, 0xb7, 0xb, 0x46, ...})
        /home/runner/work/avalanchego-internal/avalanchego-internal/snow/engine/common/queue/state.go:244 +0xf0
github.com/ava-labs/avalanchego/snow/engine/common/queue.(*JobsWithMissing).cleanRunnableStack(0xc016efd940, {0x1c08748, 0xc0000440c8})
        /home/runner/work/avalanchego-internal/avalanchego-internal/snow/engine/common/queue/jobs.go:402 +0x1e5
github.com/ava-labs/avalanchego/snow/engine/common/queue.(*JobsWithMissing).SetParser(...)
        /home/runner/work/avalanchego-internal/avalanchego-internal/snow/engine/common/queue/jobs.go:275
github.com/ava-labs/avalanchego/snow/engine/snowman/bootstrap.New({0x1c08748, 0xc0000440c8}, {{0xc013381ce0, {0x1c1c6d0, 0xc0004603f0}, 0x14, 0x1d448e93be9d1ec, {0x1c16520, 0xc00f447b00}, {0x1c22718, ...}, ...}, ...}, ...)
        /home/runner/work/avalanchego-internal/avalanchego-internal/snow/engine/snowman/bootstrap/bootstrapper.go:112 +0x5fd
github.com/ava-labs/avalanchego/chains.(*manager).createAvalancheChain(0xc0111a8380, 0xc013381ce0, {0xc01aaf80ed, 0x487ce, 0x49f13}, {0x1c1c6d0, 0xc0004603f0}, {0x1c1c6d0?, 0xc0004603f0?}, {0x7f4f885c3510, ...}, ...)
        /home/runner/work/avalanchego-internal/avalanchego-internal/chains/manager.go:888 +0x298a
github.com/ava-labs/avalanchego/chains.(*manager).buildChain(0xc0111a8380, {{0xed, 0x5f, 0x38, 0x34, 0x1e, 0x43, 0x6e, 0x5d, 0x46, ...}, ...}, ...)
        /home/runner/work/avalanchego-internal/avalanchego-internal/chains/manager.go:545 +0x10ea
github.com/ava-labs/avalanchego/chains.(*manager).createChain(0xc0111a8380, {{0xed, 0x5f, 0x38, 0x34, 0x1e, 0x43, 0x6e, 0x5d, 0x46, ...}, ...})
        /home/runner/work/avalanchego-internal/avalanchego-internal/chains/manager.go:335 +0x3d8
github.com/ava-labs/avalanchego/chains.(*manager).dispatchChainCreator(0xc0111a8380)
        /home/runner/work/avalanchego-internal/avalanchego-internal/chains/manager.go:1384 +0xc5
created by github.com/ava-labs/avalanchego/chains.(*manager).StartChainCreator
        /home/runner/work/avalanchego-internal/avalanchego-internal/chains/manager.go:1363 +0x390
```

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**
no error
**Screenshots**

**Logs**
If applicable, please include the relevant logs that indicate a problem and/or the log directory of your node. By default, this can be found at `~/.avalanchego/logs/`.

**Metrics**
If applicable, please include any metrics gathered from your node to assist us in diagnosing the problem.

**Operating System**
Which OS you used to reveal the bug.
ubuntu20.0.4
**Additional context**

https://github.com/ava-labs/avalanchego/tree/v1.10.0

**To best protect the Avalanche community security bugs should be reported in accordance to our [Security Policy](../security/policy)**

**Comment by StephenButtolph** [comment]
@justforfunya Is this repeatable every time you start the node? If so would you be able to run a patch version to verify the fix?

**Comment by StephenButtolph** [comment]
I'm going to mark this as closed - I'd still love to hear back that `v1.10.1` did resolve the issue though.
